react/finger-exercises/redux/01 - actions and reducers/auth/reducer.js

```diff
@@ -1,10 +1,21 @@
+import { actions } from './actions';
+
 const initialState = {
   email: null,
   token: null
 };
 
-function reducer(state = initialState) {
-  return state;
+function reducer(state = initialState, action) {
+  switch (action.type) {
+    case actions.LOGS_IN:
+      return {
+        ...state,
```

Comment:
Mmm for do this you should use `state.merge`. This way you merge the new param in the old state